src/libraries/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.Utils.cs

```diff
@@ -49,11 +69,18 @@ private static int Reduce(Span<uint> bits, ReadOnlySpan<uint> modulus)
 
             if (bits.Length >= modulus.Length)
             {
-                Divide(bits, modulus, default);
+                DivRem(bits, modulus, default);
 
                 return ActualLength(bits.Slice(0, modulus.Length));
             }
             return bits.Length;
         }
+
+        [Conditional("DEBUG")]
+        public static void DummyForDebug(Span<uint> bits)
```

Comment:
This needs a better name, like `InitializeForDebug`.

Alternatively it shouldn't exist at all as we typically don't do such debug vs release differences today. Any such support should rather be centralized across the .NET libraries or done automatically by the runtime.